site/content/assets.md

```diff
@@ -5,60 +5,79 @@ weight = 1
 +++
 
 Declaring assets to be processed by Trunk is simple and extensible. All assets to be processed by Trunk must follow these three rules:
+
```

Comment:
Same for this one `:)`.

Reply:
I kept my updated documentation but reverted my formatting.